.gitignore

```diff
@@ -70,3 +70,4 @@ fastlane/test_output
 .DS_Store
 Packages/*
 Package.resolved
+.swiftpm
```

Comment:
Is there a "best practice" on this yet? Like should some files within not be ignored?

Reply:
Ah: https://forums.swift.org/t/files-created-when-opening-spm-package-in-xcode-11/28589

Reply:
Also: https://github.com/github/gitignore/blob/master/Swift.gitignore#L45-L47

Reply:
It only seems to contain a workspace and some user data, so I was thinking it should always be ignored. I think I've done it on other repos too 🤔

But I haven't yet read about doing one way or the other yet...